### DISCUSSION TITLE
Alpine Error: "SyntaxError: Unexpected token '}'"

### DISCUSSION THREAD

**Discussion by pablomadariaga** [livewire/livewire#3314] [7. Bugs (archived)] [answered]
As of version 2.5.3 alpine js 2.8.2 generates this error:

`Alpine Error: "SyntaxError: Unexpected token '}'"`
`Expression: "document.getElementById("`

alpine line 133:formated

`if (console.warn(`Alpine Error: "${n}"\n\nExpression: "${t}"\nElement:`, e),`

**Comment by joshhanley** [comment, marked as answer]
@pablomadariaga a fix has been tagged for this, please upgrade to Livewire 2.5.5 and let us know if you are still having any issues 🙂

Edit: sorry your issue might actually be unrelated, but if you can upgrade and test, that'd be great thanks!

**Reply by pablomadariaga** [reply]
Thanks. everything works

**Reply by joshhanley** [reply]
Awesome!

**Reply by mfoq** [reply]
hello @pablomadariaga how did you solve the issue? only by upgrading? because I'm already using V3.5
